.readthedocs.yaml

```
@@ -0,0 +1,23 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+build:
+  os: ubuntu-20.04
```

```suggestion
  os: ubuntu-22.04
```
Maybe go with the newer LTS version?